PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/build.gradle

```diff
@@ -1,4 +1,5 @@
 intellij {
+    plugins = ['java']
```

Comment:
`azure-intellij-plugin-lib` should not depends on `java`, because it's also used by azure plugin for other ides (e.g. azure-toolkit-for-rider).